### ISSUE TITLE
Scroll position resets when posting a message

### ISSUE THREAD

**Issue by Kunsi** [element-hq/element-x-android#2562]
### Steps to reproduce

1. Open somewhat high-traffic chat room
2. Scroll up
3. Reply to message
4. Scroll position is reset to bottom, showing the just posted message

#### What did you expect?

Scroll position stays, like it happens in old Element Android or other clients

#### What happened instead?

Scroll position is reset to bottom, showing the just posted message (like in Element Web)

### Your phone model

OnePlus 7

### Operating system version

Android 12

### Application version and app store

0.4.5 (from play store)

### Homeserver

1.102.0+bookworm1 (from matrix.org debian repo)

### Will you send logs?

No

### Are you willing to provide a PR?

No

**Comment by Kunsi** [comment]
This is maybe somewhat related to #2268 and #2116

**Comment by frebib** [comment]
This was added deliberately https://github.com/element-hq/element-x-android/pull/2035

**Comment by Kunsi** [comment]
I don't understand why you would want this in a somewhat high-traffic room. I had to scroll up about 400 messages multiple times this morning (which is even more difficult with no working read markers, but that's a different issue), just because i replied to an older message. I would like to read my chats, not skip over last nights messages.

It seems to me that feature was added by someone only using Element X for 1:1 chats or very low-traffic rooms.
This behaviour should be atleast configurable, if #2035 is what users want.

**Comment by bmarty** [comment]
I agree that this is not ideal, and I have rise the concern internally.

The current behavior is sort of a workaround until we get a local echo for sending attachment from the [Rust SDK](https://github.com/matrix-org/matrix-rust-sdk/issues/1732).

There are more details about this decision https://github.com/element-hq/element-x-android/issues/1877 if it helps.

Please keep using Element Android to reply to old messages...
